src/lib/utils/logger/logger_test.go

```diff
@@ -0,0 +1,15 @@
+package logger
+
+import (
+	"testing"
+)
+
+func TestLogger(t *testing.T) {
+	file, ok := SetupLogger()
+	if !ok {
+		t.Errorf("Setting up the logger failed")
+	} else {
+		t.Logf("Logger file created")
+		t.Logf("%s", file.Name())
+	}
```

Comment:
Also, please log some messages to ensure that there is no error happenings.

Reply:
I dint get you here.

Reply:
Functions involve complex logics, if not now then later. An end-to-end test should cover the output as well. So, basically we use a normal purpose of the function which over here is logging. If the normal operation is being able to perform by the function, then the function is good.  Else, an implicit panic would be automatically returned by some low-level function.